internal/clauderig/commands/search_account_test.go

```
@@ -0,0 +1,163 @@
+// session filters. The guard runs before the flag is resolved into the scope,
+// so it has to test the flag itself — testing sc.account would let
+// `--account X --raw` through and return matches the filter never touched.
+func TestSessionScope_AccountAloneStillCountsAsNarrowing(t *testing.T) {
+	// sc.account is empty at guard time even when --account was given.
+	sc := sessionScope{}
+	accountFilter := "work"
+	if !(sc.filtering() || accountFilter != "") {
+		t.Error("--account alone must trip the raw/all guard")
+	}
+	// and with nothing set at all, it must not trip
+	if sc.filtering() || "" != "" {
```

golangci-lint (2.12.2)

[error] SA4000: identical expressions on the left and right side of the '!=' operator

(staticcheck)